.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+on:
+  # Ensure GitHub actions are not run twice for same commits
+  push:
+    branches: [master]
+    tags: ['*']
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test-node:
+    name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
```

Comment:
```suggestion
    name: Node ${{ matrix.node_version }} on ${{ matrix.os }}
```

Reply:
Thanks! yeah realized that the message ending up in checks is pretty duplicated, like this one more 👍